cmd/main.go

```diff
@@ -1,5 +1,43 @@
 package main
 
+import (
+	"context"
+	"net/http"
+	"os"
+	"os/signal"
+	"time"
+
+	"github.com/ElrondNetwork/notifier-go/config"
+	"github.com/ElrondNetwork/notifier-go/proxy"
+)
+
+var (
+	backgroundContextTimeout = 5 * time.Second
+)
+
 func main() {
+	cfg, err := config.LoadConfig()
```

Comment:
don't we want this app to support flags? you might consider the `"github.com/urfave/cli"` lib

Reply:
these will be added in a separate PR